test/orbit_web/controllers/auth_controller_test.exs

```diff
@@ -0,0 +1,53 @@
+defmodule OrbitWeb.AuthControllerTest do
+  use OrbitWeb.ConnCase
+  alias OrbitWeb.Auth.Auth
+
+  defp login_with_groups(conn, _groups) do
+    conn
+    |> get(~p"/auth/keycloak/callback?#{%{"email" => "user@example.com"}}")
+  end
+
+  describe "login_page" do
+    test "redirects to keycloak login (i.e. there isn't one)", %{conn: conn} do
```

Comment:
huh? I don't understand the parenthetical here. I was expecting something like "/ redirects to keycloak login if you're not logged in".

Reply:
This was poorly worded. I was insinuating that there is no login _page_ at all in orbit. I switched it to describe `/login` and changed the test description. 17d5e4d